packages/@winglang/sdk/src/std/array.ts

```diff
@@ -165,6 +165,16 @@ export class MutArray {
     throw new Error("Abstract");
   }
 
+  /**
+   * Removes all elements
+   *
+   * @macro for (let i = 0; i < $self$.length; i++) {delete $self$[i];} $self$.length = 0;}
```

Comment:
```suggestion
   * @macro $self$.length = 0;
```

I believe this should be enough